docs/guide/algos.rst

```diff
@@ -10,11 +10,12 @@ Name         ``Box``     ``Discrete`` ``MultiDiscrete`` ``MultiBinary`` Multi Pr
 ============ =========== ============ ================= =============== ================
 TQC          ✔️          ❌            ❌                ❌              ❌
 QR-DQN       ️❌          ️✔️            ❌                ❌              ❌
+BDPI         ❌          ✔️            ❌                ❌              ✔️
```

Comment:
it says "multiprocessing" but I only see tests with one environment in the code...
and you probably need https://github.com/DLR-RM/stable-baselines3/pull/439 to make it work with multiple envs.

Reply:
I may have mis-understood what "Multiprocessing" in this document means, and what PPO in stable-baselines is doing.

BDPI distributes its training updates on several processes, even if only one environment is used. To me, this is multi-processing, comparable to PPO that uses MPI to distribute compute. But if PPO uses multiple environments to be able to do multiprocessing, then I understand that "Multiprocessing" in the documentation means "compatible with multiple envs", not just "fast because it uses several processes".

Should I add a note, or a second column to distinguish "multiple envs" from "multi-processing with one env"?

Reply:
>To me, this is multi-processing, comparable to PPO that uses MPI to distribute compute. 

PPO with MPI distributes env and training compute (and is currently not implemented in SB3).

>then I understand that "Multiprocessing" in the documentation means "compatible with multiple envs"

yes, that's the meaning (because we can use `SubProcEnv` to distribute data collection)

>Should I add a note, or a second column to distinguish "multiple envs" from "multi-processing with one env"?

no, I think we have already enough columns in this table.